content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -150,6 +150,15 @@ pointing to the Pod's IP address. Also, Pod needs to become ready in order to ha
 record unless `publishNotReadyAddresses=True` is set on the Service.
 {{< /note >}}
 
+### Pod's hostnameFQDN field {#pod-hostnamefqdn-field}
+
+Currently, when a pod is configured to have fully qualified domain name (FQDN), the hostname field
+of the kernel contains the pod shot name. For example, if the Pod FQDN is "`busybox-1.default-subdomain.my-namespace.svc.cluster-domain.example`", the `hostname` command would return `busybox-1`, whereas the `hostname --fqdn` command would return the full FQDN. When this field is enabled by setting `hostnameFQDN: true` in the Pod spec, the full FQDN will be inserted in the hostname field of the kernel. In this case, both `hostname` and `hostname --fqdn` would return the full FQDN.
+
+{{< note >}}
+Because hostname field of the kernel is limited to 63 characters, pods that enable this feature and have a longer than 63 character FQDN will fail to start. These pods will remain in ContainerCreating with errors regarding FQDN length. One way of improving user UX is to create an [admission webhook controller](docs/reference/access-authn-authz/extensible-admission-controllers/#admission-webhooks) to control FQDN size when users create top level objects, e.g., Deployment.
```

Comment:
I presume that 63 character limit is specific to POSIX / Linux rather than to Kubernetes overall. What happens if you're running containers on (eg) Windows.

Also: [avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) such as e.g.